atlp-rwanda/Phantom-Backend-Elites
# Add express structure and config

## What does this PR do?

- It add express structure especially folders.
- It provide a welcome endpoint to test setup
- Setup API documentation with Swagger
- It provide test of an endpoint provided above.

## Description of Task to be completed?

- Creating folders for each tasks for improving code maintainability and reusability.
- Provide a welcome message to the API consumers before they start consumption.
- Setup API documentation which guides API consumers how to consume an API and guidelines for starting.
- Each task and function write to complete task needs to be tested for good practice of an API.

## How should this be manually tested?

**Clone the repository, the following command to install all depandencies.**
```
    npm install
```
or 
```
    npm i
```

- start the server with `npm start`
- Go to `http://localhost:3000/` to see welcome message
- Go to `http://localhost:3000/api-docs` to see API documentation of an api.

## Testing tools to use.

---

- [x] postman
